### DISCUSSION TITLE
How to play and stop a specified animation in AR mode?

### DISCUSSION THREAD

**Discussion by futahei** [google/model-viewer#4124] [Q&A] [answered]
Hello. I always appreciate your help with model-viewer.
I have a question about animations. While it is possible to control animations in the 3D viewer, is it not possible to do so in AR mode?
I would like to see the final state of the animation in AR after it has finished transforming. Since I could not find anything suitable in the SceneViewer intent parameters, is there anything that can be done on the model-viewer side?

**Comment by elalish** [comment, marked as answer]
As usual, only by using WebXR. Is anything blocking you from this?

**Reply by futahei** [reply]
I think WebXR is a great technology. However, my usage requires providing the same functionality on both iOS and Android. I was thinking of using animations as a way to combine 3D models of different shapes, but it seems to be difficult.

**Reply by futahei** [reply]
What I want to do is view the final state of a 3D model in AR after its animation has completed. While it would be best to achieve this with a single model, after reading the source code of model-viewer, it seems to download and load the 3D model when launching scene-viewer or quick-look. So, if I rewrite the src or ios-src just before starting AR, I can change the model to be displayed in AR, right? It would be perfect if I could revert the src to the original URL of the 3D model when AR is finished. Is it possible to do something like this?
(If my translation was incorrect, I apologize😥)

**Reply by elalish** [reply]
It might be easier by using two MV elements - the second could be hidden and used only for AR (use `activateAR()`).

**Reply by futahei** [reply]
I did not think of that! Thank you for the good idea☺
